sale_order_color_code_tintometric_system_stock/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1 @@
+* Iria Alonso <iria.alonso@solvos.es>
```

Comment:
Remember that, for `.md` files, `<` and `>` should be escaped, otherwise they won't be rendered in final `README.rst` file (as you can check for the current generated file):
```suggestion
* Iria Alonso \<iria.alonso@solvos.es\>
```